src/neo/Ledger/Blockchain.cs

```diff
@@ -310,6 +310,7 @@ private VerifyResult OnNewExtensiblePayload(ExtensiblePayload payload)
         {
             DataCache snapshot = system.StoreView;
             extensibleWitnessWhiteList ??= UpdateExtensibleWitnessWhiteList(system.Settings, snapshot);
+            if (system.RelayCache.Any(u => u is ExtensiblePayload ext && ext.Sender == payload.Sender)) return VerifyResult.PolicyFail;
```

Comment:
I'm afraid this would block valuable consensus / stateroot messages if only one message is allowed in cache

Reply:
Maybe we can add only validators and committee to the whitelist and include more in the future if it's needed?

Reply:
I think we can just remove this line.